simuleval/options.py

```diff
@@ -183,7 +189,7 @@ def general_parser():
 
 def add_slurm_args(parser):
     parser.add_argument(
-        "--slurm-partition", default="learnaccel,ust", help="Slurm partition."
+        "--slurm-partition", default="learnaccel,learnfair,seamless", help="Slurm partition."
```

Comment:
since this is a general toolkit, maybe we shouldn't set defaults here?

Reply:
Maybe we can leave seamless because we are seamless team?

Reply:
nvm, you are right. I remove the default partition.